spec/functions_spec.rb

```diff
@@ -3,11 +3,15 @@
 require 'ruby2js/filter/functions'
 
 describe Ruby2JS::Filter::Functions do
-  
-  def to_js( string)
+
+  def to_js(string)
     Ruby2JS.convert(string, filters: [Ruby2JS::Filter::Functions]).to_s
   end
-  
+
+  def to_es2016(string)
```

Comment:
At the moment, the eslevel specific tests are in spec/es201*_spec.rb.  Can this be moved to es2016_spec?  You can steal/adapt the to_js_fn(string) function from es2017_spec.

Reply:
Of course!

Reply:
Tell me is there something that should be changed, I wasn't sure if you want to have a second function or add additional filters into those tests.

Reply:
Sorry I did not listen to this part:

> You can steal/adapt the to_js_fn(string) function from es2017_spec.